scala-core-2/src/main/scala/com/baeldung/scala/await/AwaitFuture.scala

```diff
@@ -0,0 +1,54 @@
+package com.baeldung.scala.await
+
+import scala.concurrent.{ Await, Future }
+import scala.io.Source
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object AwaitFuture extends App {
+
+  // creating futures and calling a thread.sleep to wait
+  Future {
+    val contents = Source.fromURL("https://www.facebook.com").getLines().mkString
+    println(contents)
+  }
+  Thread.sleep(1000)
```

Comment:
Let's split this code up into methods. And then create a unit test file that calls the methods and runs assertions on them.